### PR TITLE
worked on header and intro section

### PR DESCRIPTION
**Pull request by MeAkib**
Hi Please check the changes. You can an idea about changes. I've modified code that isn't related to responsive.

### CODE REVIEW

File: components/Header.tsx [hunk 1]
```diff
@@ -1,45 +1,96 @@
...
+            {/* Navigation  */}
+            <nav className="">
+              <button onClick={toggleMenu} className="md:hidden block">
+                Open
+              </button>
+
+              <ul
+                className={`md:items-center items-start md:pl-0 pl-10 flex md:flex-row flex-col md:justify-start justify-center text-okwhite space-y-5 md:space-y-0 md:space-x-6 lg:space-x-10 xl:space-x-14 md:relative absolute w-full md:h-auto md:bg-oktransparent bg-okred top-0 h-screen transition-all ${
+                  toggle ? 'left-[calc(-50rem+0vw)] md:left-0' : 'left-0'
+                }`}
```
Comment (j-nolan) [inline components/Header.tsx]: Not a big deal but you can use `clsx(...)` utility here

```
className={clsx(
  'md:items-center items-start md:pl-0 pl-10 flex md:flex-row flex-col md:justify-start justify-center text-okwhite space-y-5 md:space-y-0 md:space-x-6 lg:space-x-10 xl:space-x-14 md:relative absolute w-full md:h-auto md:bg-oktransparent bg-okred top-0 h-screen transition-all',
  toggle ? 'left-[calc(-50rem+0vw)] md:left-0' : 'left-0'
)}
```

**Review by j-nolan — APPROVED** [review]
Looks fantastic thank you! I imagine you intend to re-add the sections you commented out?

You may merge this PR into `main` (let me know if it doesn't let you)

File: components/Header.tsx (hunk 1 above)
Comment (MeAkib) [reply components/Header.tsx]: I commented out sections cause I need to check responsive and those aren't responsive. I'll work one by one section

Comment (j-nolan) [reply components/Header.tsx]: Sounds good!

File: pages/index.tsx [hunk 2]
```diff
@@ -8,29 +8,44 @@ import IntroSection from '../components/IntroSection';
...
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 
 export default function Home() {
+  const [mount, setMount] = useState(false);
+
+  useEffect(() => {
+    setMount(!mount);
+  }, []);
+
```
Comment (j-nolan) [inline pages/index.tsx]: This completely cancels server-side rendering, which cancels SEO benefits. Why is it needed? Can it be done without?

Comment (MeAkib) [reply pages/index.tsx]: Yeah. it can be done without it.

**Review by j-nolan — APPROVED** [review]
Looks great, thanks Akib, could you merge into the main branch?

**Pull request merged**